src/Common/src/TypeSystem/Common/MetadataType.Interop.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Internal.TypeSystem
+{
+    public enum StringFormat
```

Comment:
It may be nice to have Interop or PInvoke in the name to make it clear what it is for.